data/scripts/actions/others/usable_mount_items.lua

```diff
@@ -0,0 +1,56 @@
+local config = {
+	[26194] = { -- vibrant egg
+		name = "vortexion",
+		mountId = 94,
+		tameMessage = "You receive the permission to ride a sparkion."
+	},
+	[26340] = { -- crackling egg
+		name = "neon sparkid",
+		mountId = 98,
+		tameMessage = "You receive the permission to ride a neon sparkid."
+	},
+	[26341] = { -- menacing egg
+		name = "vortexion",
+		mountId = 99,
+		tameMessage = "You receive the permission to ride a vortexion."
+	},
+	[25521] = { -- mysterious scroll
+		name = "rift runner",
+		mountId = 87,
+		tameMessage = "You receive the permission to ride a rift runner."
+	}
+}
+
+local usableItemMounts = Action()
+
+function usableItemMounts.onUse(player, item, fromPosition, target, toPosition, isHotkey)
+	if not player:isPremium() then
+		player:sendCancelMessage(RETURNVALUE_YOUNEEDPREMIUMACCOUNT)
+		return true
+	end
+
+	local useItem = config[item.itemid]
+	if player:hasMount(useItem.mountId) then
+		return false
+	end
+	
+	if table.contains({26194, 26340, 26341}, useItem) then
+		local storage = player:getStorageValue(PlayerStorageKeys.vortexTamer)
+		if storage == 3 then
+			player:addAchievement("Vortex Tamer")
+		else
+			player:setStorageValue(PlayerStorageKeys.vortexTamer, storage + 1)
+		end
```

Comment:
The way you get the achievement is by taming all three monsters from what I understood, now it would be enough to tame one of them three times